docs/reference/datatiers.asciidoc

```diff
@@ -37,9 +37,9 @@ TIP: The performance of an {es} node is often limited by the performance of the
 For example hardware profiles, refer to Elastic Cloud's {cloud}/ec-reference-hardware.html[instance configurations]. 
 Review our recommendations for optimizing your storage for <<indexing-use-faster-hardware,indexing>> and <<search-use-faster-hardware,search>>.
 
-IMPORTANT: {es} generally expects nodes within a data tier to share the same 
-hardware profile. Variations not following this recommendation should be 
-carefully architected to avoid <<hotspotting,hot spotting>>.
+IMPORTANT: {es} assumes nodes within a data tier share the same hardware profile (such as CPU, RAM, disk capacity) 
+as it spreads the load equally across all nodes within the data tier.
+Data tiers with unequally resourced nodes are at risk of <<hotspotting,hot spotting>>.
```

Comment:
I do not think it is true today that we spread the load equally. I also think it is not a guarantee to avoid hotspotting, which is sort of implied here.

```suggestion
.
Data tiers with unequally resourced nodes have a higher risk of <<hotspotting,hot spotting>>.
```

Reply:
++ although we do not spread the load equally, we allocate the load _assuming_ that the available resources are equal.